packages/fxa-admin-server/src/database/database.service.ts

```diff
@@ -79,8 +86,9 @@ export class DatabaseService implements OnModuleDestroy {
     this.device = Device;
     this.linkedAccounts = LinkedAccount;
 
+    this.mySqlOAuthShared = mySqlOAuthShared;
```

Comment:
Probably not a thing for this PR, but as I see this evolving, I wonder if we we should have more services broken out. Smaller services are usually more maintainable and modular. 

So perhaps we have several supporting services here that can be injected, 

- MySqlOAuthSharedService
- ConnectedServicesCacheService
- ConnectedServicesDbService